src/components/dropdown/daikin-dropdown.ts

```diff
@@ -0,0 +1,362 @@
+ *   <daikin-dropdown-item value="value3" disabled>Dropdown item 3</daikin-dropdown-item>
+ * </daikin-dropdown>
+ * ```
+ */
+@customElement("daikin-dropdown")
+export class DaikinDropdown extends LitElement {
+  static override readonly styles = css`
+    ${unsafeCSS(tailwindStyles)}
+
+    :host {
+      display: block;
+      width: 360px;
+    }
+
+    ::slotted {
+      width: 100%;
+    }
+  `;
+
+  /**
+   * Label text
+   */
+  @property({ type: String })
+  label = "";
+
+  /**
+   * Dropdown value
+   */
+  @property({ type: String })
+  value?: string;
+
+  /**
+   * Placeholder text
+   */
+  @property({ type: String })
+  placeholder = "";
+
+  /**
+   * Whether the dropdown is disabled
+   */
+  @property({ type: Boolean, reflect: true })
+  disabled = false;
+
+  /**
+   * Whether the dropdown is error
+   */
+  @property({ type: Boolean, reflect: true })
+  error = false;
+
+  /**
+   * Whether or not a drop-down menu is displayed
+   */
+  @property({ type: Boolean, reflect: true })
+  open = false;
+
+  @state()
+  private _buttonLabel = "";
+
+  @state()
+  private _buttonLabelType: "value" | "placeholder" = "placeholder";
+
+  @queryAssignedElements({ selector: "daikin-dropdown-item" })
+  private _items!: DaikinDropdownItem[];
+
+  private _buttonRef = createRef<HTMLElement>();
+
+  private _contentsRef = createRef<HTMLElement>();
+
+  private _autoUpdateCleanup: (() => void) | null = null;
+
+  private _startAutoUpdate() {
+    if (!isClient) {
+      return;
+    }
+
+    const button = this._buttonRef.value;
+    const contents = this._contentsRef.value;
+    if (!button || !contents) {
+      return;
+    }
+
+    // TODO(DDS-1226): refactor here with Popover API + CSS Anchor Positioning instead of using floating-ui
+    this._autoUpdateCleanup?.();
+    this._autoUpdateCleanup = autoUpdate(button, contents, () => {
+      computePosition(button, contents, {
+        placement: "bottom",
+        middleware: [
+          flip({ fallbackStrategy: "initialPlacement" }),
+          offset({ mainAxis: -1 }),
+        ],
+      })
+        .then(({ x, y }) => {
+          Object.assign(contents.style, {
+            left: `${x}px`,
+            top: `${y}px`,
+          });
+        })
+        .catch((e: unknown) => console.error(e));
+    });
+  }
+
+  private _uninstallAutoUpdate() {
+    this.open = false;
+    this._autoUpdateCleanup?.();
+    this._autoUpdateCleanup = null;
+  }
+
+  private _reflectButtonLabel() {
+    const items = this._items;
+
+    const itemIndex: number = items.findIndex(
+      ({ value }) => this.value === value
+    );
+
+    if (itemIndex > -1) {
+      for (const item of items) {
+        item.selected = false;
+      }
+      this._buttonLabel = items[itemIndex].textContent ?? "";
+      this._buttonLabelType = "value";
+      items[itemIndex].selected = true;
+    } else {
+      this._buttonLabel = this.placeholder;
+    }
```

Comment:
1. `selected` is not updated if `itemIndex < 0`.
2. `_buttonLabelType` is not updated if `itemIndex < 0`.
3. I think it is preferable to have a button label type that indicates whether or not the currently selected item is available - From this perspective, it would be more natural to refer to the `this.placeholder` directly in the `render`, rather than using `this._buttonLabel`.

```suggestion
    const selectedItem = items.find(
      ({ value }) => this.value === value
    );

    for (const item of items) {
      item.selected = item === selectedItem;
    }

    this._hasSelectedItem = !!selectedItem;
    this._selectedItemLabel = selectedItem?.textContent ?? "";
```